src/main/java/net/sf/jabref/gui/journals/AbbreviationsFileViewModel.java

```diff
@@ -0,0 +1,136 @@
+ * It actually doesn't save the files as objects but rather saves
+ * their paths. This also allows to specify pseudo files as placeholder objects.
+ */
+public class AbbreviationsFileViewModel {
+
+    private final SimpleListProperty<AbbreviationViewModel> abbreviations = new SimpleListProperty<>(
+            FXCollections.observableArrayList());
+    private final ReadOnlyBooleanProperty isBuiltInList;
+    private final String name;
+    private final Optional<Path> path;
+
+
+    public AbbreviationsFileViewModel(String filePath) {
+        this.path = Optional.ofNullable(Paths.get(filePath));
+        this.name = path.get().toAbsolutePath().toString();
+        this.isBuiltInList = new SimpleBooleanProperty(false);
+        this.abbreviations.add(new AbbreviationViewModel(null));
+    }
+
+    /**
+     * This constructor should only be called to create a pseudo abbreviation file for built in lists.
+     * This means it is a placeholder and it's path will be null meaning it has no place on the filesystem.
+     * It's isPseudoFile property will therefore be set to true.
+     */
+    public AbbreviationsFileViewModel(List<AbbreviationViewModel> abbreviations, String name) {
+        this.abbreviations.addAll(abbreviations);
+        this.name = name;
+        this.path = Optional.empty();
+        this.isBuiltInList = new SimpleBooleanProperty(true);
+    }
+
+    public void readAbbreviations() throws FileNotFoundException {
+        if (path.isPresent()) {
+            List<Abbreviation> abbreviationList = JournalAbbreviationLoader
+                    .readJournalListFromFile(path.get().toFile());
+            abbreviationList.forEach(abbreviation -> abbreviations.addAll(new AbbreviationViewModel(abbreviation)));
+        } else {
+            throw new FileNotFoundException();
+        }
+    }
+
+    /**
+     * This method will write all abbreviations of this abbreviation file to the file on the file system.
+     * It essentially will check if the current file is a built in list and if not it will call
+     * {@link AbbreviationWriter#writeOrCreate()}.
+     *
+     * @throws IOException
+     */
+    public void writeOrCreate() throws IOException {
+        if (!isBuiltInList.get()) {
+            List<Abbreviation> actualAbbreviations = Lists.newArrayList();
+            abbreviations.forEach(abb -> {
+                if (!abb.isPseudoAbbreviation()) {
+                    actualAbbreviations.add(abb.getAbbreviationObject());
+                }
+            });
+            AbbreviationWriter.writeOrCreate(path.get(), actualAbbreviations, StandardCharsets.UTF_8);
+        }
+    }
+
+    public SimpleListProperty<AbbreviationViewModel> abbreviationsProperty() {
+        return this.abbreviations;
+    }
+
+    public boolean exists() {
+        return path.isPresent() && Files.exists(path.get());
+    }
+
+    public String getAbsolutePath() {
+        return path.get().toAbsolutePath().toString();
```

Comment:
Be careful. The path may be an empty optional. I would propose to change the return value to Optional<String> and write `return path.map(toAbsolutePath().toString());`